packages/documentation/src/content/docs/integration/event-handlers.mdx

```diff
@@ -75,6 +77,18 @@ Example: An outgoing payment for $12 has been created.
 `}
 />
 
+Example: Cancel outgoing payment due to low funds
+
+<Mermaid
+  graph={`sequenceDiagram
+    participant ASE as Account Servicing Entity
+    participant R as Rafiki
+    R->>ASE: webhook event: outgoing payment created,<br>debit amount: $12
+    ASE->>ASE: check if account has enough balance
+    ASE->>R: admin API call: CancelOutgoingPayment<br>reason: Not enough balance
+`}
```

Comment:
we could probably avoid having to have a second diagram by using `alt` to describe the two possible flows in the original diagram